### PR TITLE
chore: update PR template with copilot

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    -       |
| 🇨🇳 Chinese |    -       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at ebb6388</samp>

This pull request adds a `
### Summary
📝🌐👩‍💻

This pull request adds two templates for copilot to generate code review messages in English and Chinese for the ant-design project. The templates are in `.github/PULL_REQUEST_TEMPLATE.md` and `.github/PULL_REQUEST_TEMPLATE/pr_cn.md`. The goal is to improve the code review process and the project's accessibility.

> _`copilot` writes_
> _code review messages for us_
> _autumn of coding_

### Walkthrough
* Add templates for copilot to generate code review messages in English and Chinese ([link](https://github.com/ant-design/ant-design/pull/41529/files?diff=unified&w=0#diff-18813c86948efc57e661623d7ba48ff94325c9b5421ec9177f724922dd553a35R64-R78), [link](https://github.com/ant-design/ant-design/pull/41529/files?diff=unified&w=0#diff-01e0405c37ba25be0b9a335536626378e8defa04d3285eac7439ef984058ffa4R64-R77))

` command to the pull request templates for both English and Chinese versions, which enables the GitHub Copilot AI assistant to review and suggest improvements for the code in the pull request. This change aims to enhance the code quality and reliability of the repository.

### <samp>🤖 Generated by Copilot at ebb6388</samp>

*  Add Copilot command to pull request templates for English and Chinese versions ([link](https://github.com/ant-design/ant-design/pull/41529/files?diff=unified&w=0#diff-18813c86948efc57e661623d7ba48ff94325c9b5421ec9177f724922dd553a35R64-R67), [link](https://github.com/ant-design/ant-design/pull/41529/files?diff=unified&w=0#diff-01e0405c37ba25be0b9a335536626378e8defa04d3285eac7439ef984058ffa4R64-R67))

### CODE REVIEW

**Comment by afc163** [comment]
这个 PR 自举一下？

**Review by li-jia-nan — APPROVED** [review]

**Comment by afc163** [comment]
copilot:all 用中文

**Comment by MadCcc** [comment]
> 这个 PR 自举一下？

template 合进去才能生效吧，先手动改了

**Comment by afc163** [comment]
copilot:all

File: .github/PULL_REQUEST_TEMPLATE/pr_cn.md [hunk 1]
```diff
@@ -62,6 +62,16 @@
...
+以下为 copilot 自动生成的 CR 结果，请勿修改
+-->
+
+### 🚀 概述
+
+copilot:summary
+
+### 🔍 实现细节
+
+copilot:walkthrough
```
Comment (afc163) [inline .github/PULL_REQUEST_TEMPLATE/pr_cn.md:77]: 放在 📝 Changelog 前面？

**Review by afc163 — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE/pr_cn.md (hunk 1 above)
Comment (MadCcc) [reply .github/PULL_REQUEST_TEMPLATE/pr_cn.md:77]: 是想的把自动生成的放在最后面，这样填的时候就不用关心是自动的还是需要手动改的了

Comment (MadCcc) [reply .github/PULL_REQUEST_TEMPLATE/pr_cn.md:77]: 不然还会想这里是不是改手动写点东西

**Pull request merged**